src/features/alerts/components/table-alerts-empty-state.tsx

```diff
@@ -0,0 +1,40 @@
+import { Heading } from "@stacklok/ui-kit";
+import { ReactNode } from "react";
+import { tv } from "tailwind-variants";
+import { useAlertsEmptyState } from "../hooks/use-alerts-empty-state";
+
+const actionsStyle = tv({
+  base: "mx-auto mt-6",
+  variants: {
+    actions: {
+      1: "",
+      2: "grid grid-cols-2 gap-2",
+    },
+  },
+});
+
+export function Actions({ actions }: { actions: [ReactNode, ReactNode?] }) {
+  return (
+    <div className={actionsStyle({ actions: actions.length })}>{actions}</div>
+  );
+}
+
+export function TableAlertsEmptyState({ isLoading }: { isLoading: boolean }) {
```

Comment:
look at this suggestion: https://github.com/stacklok/codegate-ui/pull/239/files#r1937422209

I think that this component could be a completely generic one, and probably *should* in order to encourage the reuse of the same pattern :thinking: 

It feels weird to bind this very generic representation logic (with all actual content being parametric) with a very specific logic in the `useAlertsEmptyState` hook. Even if you prefer the hook method, I would not bind it to this component, it could be called in a more alerts table-specific context